discatpy/bot.py

```diff
@@ -15,6 +15,31 @@
 
 
 class Intents(Flag):
+    if t.TYPE_CHECKING:
+        def __init__(
+            self,
+            guilds: bool = ...,
```

Comment:
These parameters are not keyword forced.

Reply:
[`b9449ce` (#18)](https://github.com/discatpy-dev/library/pull/18/commits/b9449ce65dede1ffe8bd714334dfb47f61533a39)